### PR TITLE
update tispark document for tispark-2.1.1

### PR DESCRIPTION
**Pull request by marsishandsome**
### What is changed, added or deleted? 
update tispark document for tispark-2.1.1

### Which version does your change affect? 
v3.0 & dev

### CODE REVIEW

**Comment by marsishandsome** [comment]
@birdstorm @zhexuany @ilovesoup should we rename `tispark-quick-start-guide_v1.x.md` to `tispark-quick-start-guide.md`

**Comment by birdstorm** [comment]
@marsishandsome I think we should.

File: v3.0/reference/tispark.md [hunk 1]
```diff
@@ -76,14 +73,14 @@ spark.sql.extensions org.apache.spark.sql.TiExtensions
 
 ## 部署 TiSpark
 
-TiSpark 的 jar 包可以在[这里](http://download.pingcap.org/tispark-latest-linux-amd64.tar.gz)下载，解压并拷贝到合适的目录。
+TiSpark 的 jar 包可以在[这里](https://github.com/pingcap/tispark/releases)下载对应版本的jar包并拷贝到合适的目录。
```
Comment (zhexuany) [inline v3.0/reference/tispark.md]: 中英文之间需空格

**Comment by marsishandsome** [comment]
@dcalvin PTAL

File: v3.0/reference/tispark.md (hunk 1 above)
Comment (marsishandsome) [reply v3.0/reference/tispark.md]: fixed

**Pull request merged**